code/modules/overmap/ships/_ships.dm

```diff
@@ -191,7 +191,6 @@
 	var/turf/T = locate(nx,ny,z)
 	if(T)
 		forceMove(T)
-
```

Comment:
💢

Reply:
cant remember when i even messed with this line, fixed.